pantavisor.c

```diff
@@ -961,6 +988,9 @@ static pv_state_t pv_shutdown(struct pantavisor *pv, shutdown_type_t t)
 
 static pv_state_t _pv_reboot(struct pantavisor *pv)
 {
+	if (pv_debug_start_timeout_shell()) {
```

Comment:
Another question. In this case, we are deferring the reboot, how does PV_STATE_WAIT it has to get back to PV_STATE_REBOOT?